build.xml

```diff
@@ -527,6 +528,27 @@
         <phingcall target="twig-lint"/>
     </target>
 
+    <target name="upgrade-backend" description="Creates empty file for backend upgrade notes.">
+        <exec executable="php">
+            <arg value="-r"/>
+            <arg value="copy('upgrade-notes/_template.md', 'upgrade-notes/backend_' . date('Ymd_His') . '.md');"/>
+        </exec>
+    </target>
+
+    <target name="upgrade-storefront" description="Creates  empty file named for storefront upgrade notes.">
+        <exec executable="php">
+            <arg value="-r"/>
+            <arg value="copy('upgrade-notes/_template.md', 'upgrade-notes/storefront_' . date('Ymd_His') . '.md');"/>
+        </exec>
+    </target>
```

Comment:
What about a single phing target that asks about the type? 

Something like 

```
<target name="upgrade-generate">
        <input message="Which one?"
               propertyName="upgradeType"
               validArgs="backend,storefront,both"
        />

        <if>
            <or>
                <equals arg1="${upgradeType}" arg2="backend"/>
                <equals arg1="${upgradeType}" arg2="both"/>
            </or>
            <then>
                ... generate backend ...
            </then>
        </if>

        <if>
            <or>
                <equals arg1="${upgradeType}" arg2="storefront"/>
                <equals arg1="${upgradeType}" arg2="both"/>
            </or>
            <then>
                ... generate storefront ...
            </then>
        </if>
    </target>
    ```

Reply:
That is very nice, rewrote it as such.